src/index.html

```diff
@@ -1,12 +1,451 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+
+  <body class="page__body">
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img
+          src="./images/logo.png"
+          alt="moyo_logo"
+        >
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__item">
+            <a
+              href="#apple"
+              class="header__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#samsung"
+              class="header__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#smartphones"
+              class="header__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#laptops&computers"
+              class="header__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#gadgets"
+              class="header__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#tablets"
+              class="header__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#photo"
+              class="header__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#video"
+              class="header__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div class="card" data-qa="card">
```

Comment:
Follow the BEM, please 
```suggestion
      <div class="catalog__card card" data-qa="card">
```

Reply:
Ok